mkdocs.yml

```
@@ -97,9 +97,9 @@ nav:
   - Home: 
     - index.md
 
-  - Getting Started: 
+  - Get Started on NCI: 
     - getting_started/index.md
-    - First Steps: getting_started/first_steps.md
+    - Set Up and Customize your NCI Account: getting_started/first_steps.md
```

```suggestion
    - Set Up your NCI Account: getting_started/first_steps.md
```

See a few comments above